userguide/sharing.rst

```diff
@@ -197,8 +199,8 @@ information given when the share was created.
    |                              |               |          | automounting or volumes created by a preexec script.                                                          |
    |                              |               |          |                                                                                                               |
    +------------------------------+---------------+----------+---------------------------------------------------------------------------------------------------------------+
-   | AFP3 UNIX Privs              | checkbox      | ✓        | Enable Unix privileges supported by OSX 10.5 and higher. Do not enable this if the network contains           |
-   |                              |               |          | macOS 10.4 clients or lower as they do not support this feature.                                              |
+   | AFP3 UNIX Privs              | checkbox      | ✓        | Enable Unix privileges supported by Mac OS X 10.5 and higher. Do not enable this if the network contains           |
```

Comment:
s/Do not enable this if/Do not enable if/
s/contains/has/

Reply:
Again, I did a find/replace of "Mac OS X" spellings here, nothing more.  I'm a first time contributor to this project, minor style changes like that should be made by someone that knows the project's style rules...

Reply:
Sorry, just wanted to be thorough.